content/blog/2022/2022-04-20-KaplanMeier.md

```diff
@@ -0,0 +1,73 @@
+
+Perhaps the easiest way of getting an understanding of the concepts involved in the estimation of a survival function is with a simple example. Say we get our hands on a tiny dataset from a not-so-grim survival scenario: 10 people received 10 flowers for their birthday and tracked when the flowers wither. The tracking lasted for 10 days. Some flowers wither quicker than others and we are interested in the probability of a flower withering over time. The two main parameters of interest are thus time (in days) and flower withering-status, where "status = 1" means withered and "status = 0" means not-withered. How are we to go about determining the probability of withering over time?
+
+![](/orange-hugo/static/blog_img/2022/2022-04-20-flowerdata.png)
+
+There are two distinct problems when it comes to the analysis of survival:
+ 1. The first one has to do with the fact that "status = 0" does not necessarily mean that the flower has survived. What it does mean is that from that moment on the flower is no longer part of the ongoing study. For instance a cat might have knocked over the vase and the flower was then discarded along with the broken vase. This does not mean the flower has withered nor that it has survived. We do not want to discard these kinds of data instances, so instead we label them as **censored**. This is what "status = 0" really stands for.
+ 2. The second problem has to do with time. First of, we do not necessarily have status data available for each day and indeed in our flower dataset we do not - we only have status data for 7 out of 10 days. Furthermore the effects (for instance, of being without a root) accumulate over time. Even though a flower might have survived for 5 days, it's survival probability at day 5 is not 100 %. A decrease of the survival probability also accumulates over time, so it cannot be determined by simply counting the survivors and non-survivors at a particular day. 
+ 
+So which statistical tool should we use? 
+
+Although the fact that we have a numeric variable, time, might tempts us to use linear regression this will not be very useful since we just mentioned that the survival probability is non-linear. We might try logistic regression since the probability curve produced by it is not linear and might catch the trend of survival over time better. But logistic regression won't do either. The fifth flower has a status 0 on day 3. As we said before, this does not mean the flower has withered - it only means something happened to it that prevented it from continuuing being part of the study. The probabilities in logistic regression would count this flower as if it survived after the third day, simply because it's status it not 1. Thus, logistic regression overestimates survival probability and is therefore also an inappropriate tool to analyze survival data.
```

Comment:
might tempts us --> might tempt us
continuuing being part --> continuing to be a part
because it's status --> because its status